### ISSUE TITLE
[ADE Support]: Core Veteran Experience Crew: 508 Audit Report for 44897 Veteran Status Card Findings - Defect 2 | Critical

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#117798]
> [!NOTE]
> **QUICK SUMMARY:** _Parent ticket of #117682 ._

### Defect 2 | Critical | Created:
_WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text._

On the Veteran Status Card | Veterans Affairs screen (View your Veteran Status Card), the heading structure cannot be understood by assistive technology. The Frequently asked questions heading and the subordinate content, such as What if my Veteran Status Card displays incorrect information?, are read as level two headings. (New: 08/12/2025)

On the Types Of Veteran ID Cards | Veterans Affairs screen (View your Veteran Status Card -> What other types of Veteran ID are available? -> Learn about other types of Veteran ID cards), the region structure cannot be understood by assistive technology—there are two Secondary navigation regions. (New: 08/12/2025)

## Supporting Artifacts 
- [Project Site](https://staging.va.gov/profile)

### Ticket Updates
- [x] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [x] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [x] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [x] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by coforma-jamie** [comment]
Feedback in parent, closing
